CONTRIBUTING.md

```diff
@@ -282,3 +284,57 @@ By making a contribution to this project, I certify that:
   personal information I submit with it, including my sign-off) is
   maintained indefinitely and may be redistributed consistent with
   this project or the open source license(s) involved.
+
+#### LTS contributions
+
+Before backporting or contributing to the LTS releases make sure you are
+familiar with the LTS information and schedules at
+https://github.com/nodejs/LTS#lts-schedule
+
+Items may be selected as candidates for inclusion into the "active LTS"
+branch(es), especially if they are bug fixes.  This will be at the
+discretion of the person releasing that new version by default, but
+anyone can request a backport by adding the label "its-watch-vX.x" (e.g. 
```

Comment:
Well, anyone can request that a collaborator add the `lts-watch-vX.x` or `dont-land-on-vX.x` labels by adding a suitable comment.

Also "its-watch-vX.x" -> `lts-watch-vX.x` ( `i` -> `l` and use backticks) for consistency.

We should also probably link to https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md#how-can-i-help